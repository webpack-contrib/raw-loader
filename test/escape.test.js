/* eslint-disable
  prefer-destructuring,
*/
import webpack from './helpers/compiler';

describe('Loader', () => {
  test('Escape', async () => {
    const config = {
      loader: {
        test: /\.txt/,
        options: {},
      },
    };

    const stats = await webpack('escape/fixture.js', config);
    const { source } = stats.toJson().modules[1];

    expect(source).toMatchSnapshot();
  });
});
