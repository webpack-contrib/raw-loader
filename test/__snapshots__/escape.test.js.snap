// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Escape 1`] = `"export default \\"\\\\\\\\u2029 Lorem ipsum dolor sit amet, \\\\\\\\u2028\\\\n\\""`;
